tox.ini

```diff
@@ -100,6 +100,7 @@ description = run tests on latest version of all dependencies
 passenv = {[global_var]passenv}
 extras = test
 deps =
+    rich
```

Comment:
I prefer to not add it as an "official" optional dependency in the pyproject.toml but prefer to still test that all test run fine if rich is installed.